extra-web/skypeforlinux/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME="skypeforlinux"
+PKGSEC="web"
+PKGDEP="gtk-2 alsa-lib gconf libgnome-keyring nss"
+PKGDES="An application that provides video chat and voice call services"
+
```

Comment:
Add `FAIL_ARCH="!(amd64)"` here, as `amd64` is the only architecture that could use this package.